lib/server/services/lhcPeriod/LhcPeriodStatisticsService.js

```diff
@@ -28,15 +28,28 @@ const { NotFoundError } = require('../../errors/NotFoundError');
  * but this breaks limit clause, so pagination.
  * There is related issue for reference: https://github.com/sequelize/sequelize/issues/11983
  */
-const DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE = [
-    `(SELECT GROUP_CONCAT(DISTINCT(r.lhc_beam_energy))
+const DISTINCT_ENERGIES_IN_STATISTICS_ATTRIBUTE = {
+    query: `(SELECT GROUP_CONCAT(DISTINCT(r.lhc_beam_energy))
         FROM runs AS r
         WHERE r.lhc_period_id = lhcPeriod.id 
             AND r.definition = 'PHYSICS'
     )`,
 
-    'distinctEnergies',
-];
+    alias: 'distinctEnergies',
+};
+
+const PDP_BEAM_TYPES_IN_STATISTICS_ATTRIBUTE = {
+    query: `(SELECT r.pdp_beam_type
+        FROM runs AS r
+        WHERE r.lhc_period_id = lhcPeriod.id 
+            AND r.definition = 'PHYSICS'
+            AND r.pdp_beam_type IS NOT NULL
+            AND r.pdp_beam_type != 'cosmic'
+        LIMIT 1
```

Comment:
Why limit 1 ?

Reply:
It's assumed that lhc_period is period of some common parameters of runs (PHYSICS), e.g. the same beam type. If you see data in db, indeed, all PHYSICS runs belonging to the one lhc periods have the same pdp_beam_type.
Because some might be null (errors, whatever), there is `NOT NULL` condition
Because need only one portion of information about beam type, there is `LIMIT 1`